astrobase/services/tesslightcurves.py

```diff
@@ -222,14 +222,36 @@ def get_hlsp_lightcurves(tic_id,
     return return_lcfiles
 
 
-def get_eleanor_lightcurves(tic_id, download_dir=None):
+def get_eleanor_lightcurves(tic_id, download_dir=None, targetdata_kwargs=None):
```

Comment:
If you want to make this simpler, you can use:

```
def get_eleanor_lightcurves(tic_id, download_dir=None, **targetdata_kwargs):
```

then pass them to eleanor using:

```
eleanor.TargetData(star, **targetdata_kwargs)
```

below, but that's a bit magicky, so perhaps not the best approach.

Reply:
Yeah. Realized, and avoided because of the magicky thing ;-)